### ISSUE TITLE
potential Yatai regression in 0.3.10

### ISSUE THREAD

**Issue by yubozhao** [bentoml/Yatai#266]
Issue:

```
Here is my test:
Install Yatai 0.3.8 with local DB --> Push / Build / Deploy : OK
Upgrade helm release to 0.3.9 --> Push / Build / Deploy: OK
Upgrade helm release to 0.3.10 --> Push / Build / deploy: OK
Then I want to use external database on AZURE so I update the values of helm chart and upgrade release.
As this is a new DB, I need to recreate a user + token
Push / Build: OK
The problem is that I cannot create new deployment anymore, the status is building and no pods are created in the yatai namespace...
Here are the output of the controller:
W0705 17:14:22.142726       1 reflector.go:324] pkg/mod/k8s.io/client-go@v0.24.0-beta.0/tools/cache/reflector.go:167: failed to list *v1.CSIDriver: csidrivers.storage.k8s.io is forbidden: User "system:serviceaccount:yatai-components:yatai-yatai-deployment-operator" cannot list resource "csidrivers" in API group "storage.k8s.io" at the cluster scope
E0705 17:14:22.142762       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.24.0-beta.0/tools/cache/reflector.go:167: Failed to watch *v1.CSIDriver: failed to list *v1.CSIDriver: csidrivers.storage.k8s.io is forbidden: User "system:serviceaccount:yatai-components:yatai-yatai-deployment-operator" cannot list resource "csidrivers" in API group "storage.k8s.io" at the cluster scope
1.6570412863266902e+09	INFO	start cleaning up abandoned runner services	{"func": "doCleanUpAbandonedRunnerServices"}
```

Updates:
1. created an issue: https://github.com/bentoml/Yatai/issues/266

**Comment by mts-dyt** [comment]
Hi,

Actually it seems that this is not related to Yatai version but to the component `yatai-yatai-deployment-operator`  in namespace `yatai-component`
Yatai `0.3.10` force the image to `quay.io/bentoml/yatai-deployment-operator:v0.7.0` and this lead to the error reported. If I manually get back to `quay.io/bentoml/yatai-deployment-operator:v0.6.0`, deployments are OK.

I managed to reproduce the same error on the previous Yatai version by upgrading the yatai component `deployment` to version `0.9.0` directly in the WebUI.

**Comment by yetone** [comment]
@mts-dyt I fixed this in Yatai `v0.3.11`. You can upgrade Yatai first and then go to the Yatai components page (`/clusters/default/yatai_components`) in the Yatai UI and upgrade the `deployment` component to `v0.10.0`
